pkg/controller/controller.go

```diff
@@ -1283,6 +1283,9 @@ func (p *csiProvisioner) getSecretsFromSC(ctx context.Context, volume *v1.Persis
 			}
 
 			// Resolve provision secret credentials.
+			if volume.Spec.ClaimRef == nil {
```

Comment:
If we dont have the secrets, dont we need to allow deletion? lets the CSI driver try to handle it

Reply:
Actually, `action-runner-controller` deletes PV's  ClaimRef intentionally to make PV orphan and reuse.
So, i think we should leave this PV as orphan

Reply:
Yes, that is for the `ReclaimPolicy=Retained` PV right, not for the `ReclaimPolicy=Delete`?

Reply:
Oh.. you are right! @Madhu-1 
I'll follow your suggestion :) thx!!

Reply:
Change LGTM, you need review or approval from @msau42 @jsafrane